src/test/resources/PhonyClass.java

```diff
@@ -27,7 +27,7 @@ public void method2() {
      * This should not trigger the @todo parser becauase it's not an actual todo.
      */
     public void method3() {
-        "// @todo #187:30min this should not trigger the todo parser because this is not an actual todo".trim();
+        "// @\\todo #187:30min this should not trigger the todo parser because this is not an actual todo".trim();
```

Comment:
since the parser is language agnostic, this case will fail, cause is not recognizing `trim()`. Test is fixed by "escaping" the todo